tensor2tensor/data_generators/generator_utils.py

```diff
@@ -22,13 +22,12 @@
 import io
 import os
 import tarfile
-import urllib
 
 # Dependency imports
 
 import six
 from six.moves import xrange  # pylint: disable=redefined-builtin
-import six.moves.urllib_request
+import six.moves.urllib_request # Imports urllib on Python2, urllib.request on Python3
```

Comment:
This doesn't seem to work for us with python2 at Google. Could you try this:
import six.moves.urllib_request as urllib